app/src/main/java/com/powerhungers/fomezero/base_app/BaseApplication.kt

```diff
@@ -15,9 +17,10 @@ class BaseApplication : Application() {
                 listOf(
```

Comment:
```suggestion
                listOf(
                       dataRemoteModule,
                       dataLocalModule,
                       loginModule,
                       registrationModule,
                )
```
o que acha de organizar na mesma ordem dos pacotes do projeto